sw/device/silicon_creator/rom_ext/BUILD

```diff
@@ -210,6 +210,7 @@ cc_library(
     name = "rom_ext",
     srcs = [
         "rom_ext.c",
+        "rom_ext_immutable.c",
```

Comment:
Is this now a mandatory part of the ROM_EXT?

Reply:
Oops, no, we can remove this. The E2E test utilizes the OTTF running at the ROM_EXT stage. We don't need to add anything to the ROM_EXT until we want to. Fixed.